src/lib/styles/styles.scss

```diff
@@ -12,7 +12,7 @@
 @import '../components/forms/input/date-picker/terra-date-picker.component.glob';
 @import '../components/file-browser/terra-file-browser.component.glob';
 @import '../components/forms/input/terra-input.component.glob';
-
+@import '../components/table/table-column-settings/dialog/table-settings-dialog.glob';
```

Comment:
Globale Styles? Muss das wirklich sein?
In den Beispielen geht das auch mit ViewEncapsulation.

Reply:
die greifen dann nicht